packages/graphql-language-service-server/package.json

```diff
@@ -52,6 +52,7 @@
     "nullthrows": "^1.0.0",
     "vscode-jsonrpc": "^8.0.1",
     "vscode-languageserver": "^8.0.1",
+    "vscode-languageserver-types": "^3.17.2",
```

Comment:
added as dependency since imported in https://github.com/graphql/graphiql/blob/6c3ca902d769893a4201f8856299af18b7e21a1b/packages/graphql-language-service-server/src/GraphQLLanguageService.ts#L48-L52